mne/time_frequency/psd.py

```diff
@@ -7,15 +7,15 @@
 from ..parallel import parallel_func
 from ..io.proj import make_projector_info
 from ..io.pick import pick_types
-from ..utils import logger, verbose, _time_mask
+from ..utils import logger, verbose, deprecated, _time_mask
 
 
 @verbose
+@deprecated('This will be deprecated in release v0.13, see psd_ functions.')
 def compute_raw_psd(raw, tmin=0., tmax=None, picks=None, fmin=0,
                     fmax=np.inf, n_fft=2048, n_overlap=0,
                     proj=False, n_jobs=1, verbose=None):
     """Compute power spectral density with average periodograms.
-
```

Comment:
I realized what it was - I think I was copy/pasting some old code from github and I guess it doesn't include the extra lines :P